.github/workflows/arm-bicep-e2e-windows.yaml

```diff
@@ -0,0 +1,137 @@
+        run: task deps:download
+
+      - name: 🔨 Setup Test tools
+        run: task test:tools
+
+      - name: Install Bicep
+        shell: powershell
+        run: |
+          # Download and install Bicep for Windows
+          $bicepUrl = "https://github.com/Azure/bicep/releases/latest/" +
+            "download/bicep-win-x64.exe"
+          $bicepPath = "C:\Program Files\Azure Bicep CLI"
+          $bicepExe = "$bicepPath\bicep.exe"
+
+          # Create directory if it doesn't exist
+          New-Item -ItemType Directory -Force -Path $bicepPath | Out-Null
+
+          # Download Bicep
+          Invoke-WebRequest -Uri $bicepUrl -OutFile $bicepExe
+
+          # Add to PATH for this session
+          $env:PATH += ";$bicepPath"
+          echo "$bicepPath" | Out-File -FilePath $env:GITHUB_PATH `
+            -Encoding utf8 -Append
+
+          # Verify installation
+          & $bicepExe --version
+
+      - name: Install Azure CLI
+        shell: powershell
+        run: |
+          # Download and install Azure CLI for Windows
+          $azCliUrl = "https://aka.ms/installazurecliwindows"
+          $azCliPath = "$env:TEMP\AzureCLI.msi"
+          
+          # Download Azure CLI MSI
+          Invoke-WebRequest -Uri $azCliUrl -OutFile $azCliPath
+          
+          # Install Azure CLI
+          Start-Process msiexec.exe -Wait -ArgumentList "/I $azCliPath /quiet"
+          
+          # Add Azure CLI to PATH for this session
+          $env:PATH += ";C:\Program Files (x86)\Microsoft SDKs\Azure\CLI2\wbin"
+          echo "C:\Program Files (x86)\Microsoft SDKs\Azure\CLI2\wbin" | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
+          
+          # Verify installation
+          az --version
+
+      - name: 'Az CLI login - federated'
+        uses: azure/login@v2
+        with:
+          client-id: ${{ secrets.AZURE_OID_CLIENT_ID }}
+          tenant-id: ${{ secrets.MPF_TENANTID }}
+          subscription-id: ${{ secrets.MPF_SUBSCRIPTIONID }}
+
+      - name: 🧪 Run ARM CLI Tests
+        shell: powershell
+        run: |
+          $output = go run ./cmd arm --templateFilePath ./samples/templates/aks-private-subnet.json --parametersFilePath ./samples/templates/aks-private-subnet-parameters.json
+          $lineCount = ($output | Measure-Object -Line).Lines
+          Write-Host "Result line count: $lineCount"
+          if ($lineCount -ne 14) {
+            Write-Host "Expected 14 lines, got $lineCount"
+            exit 1
+          }
+          Write-Host "Test passed: $lineCount lines"
+
+      - name: 🧪 Run Bicep CLI Tests
+        shell: powershell
+        run: |
+          $output = go run ./cmd bicep --bicepFilePath ./samples/bicep/aks-private-subnet.bicep --parametersFilePath ./samples/bicep/aks-private-subnet-params.json
+          $lineCount = ($output | Measure-Object -Line).Lines
+          Write-Host "Result line count: $lineCount"
+          if ($lineCount -ne 14) {
```

Comment:
Due to difference between powershell and bash processing of newlines the expected count should be 13 and not 14